### PR TITLE
Fix leftover typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via codespell

### CODE REVIEW

File: projectforge-business/src/main/resources/flyway/migrate/common/V7.4.1.0__7.4.1.0-RELEASE-VACATION_OTHER_REPLACEMENTS.sql [hunk 1]
```diff
@@ -1,9 +1,9 @@
--- Vacation forms of employees supports now others substitutes (replacements) in addition to the main substitue (replacement)
+-- Vacation forms of employees now supports others substitutes (replacements) in addition to the main substitute (replacement)
 
 CREATE TABLE t_employee_vacation_other_replacements
 (
     vacation_id INTEGER NOT NULL,
-    employee_id     INTEGER NOT NULL
+    employee_id INTEGER NOT NULL
 );
```
Comment (luzpaz) [inline projectforge-business/src/main/resources/flyway/migrate/common/V7.4.1.0__7.4.1.0-RELEASE-VACATION_OTHER_REPLACEMENTS.sql:7]: Please examine source change

Comment (kreinhard) [reply projectforge-business/src/main/resources/flyway/migrate/common/V7.4.1.0__7.4.1.0-RELEASE-VACATION_OTHER_REPLACEMENTS.sql:7]: Can't change this. It's a flyway migration script and can't be changed afterwards (due to hashsum).

**Review by kreinhard — CHANGES_REQUESTED** [review]
resources/flyway/**.sql can't be modified (due to flyway hashcodes).

File: projectforge-business/src/main/resources/flyway/migrate/common/V7.4.1.0__7.4.1.0-RELEASE-VACATION_OTHER_REPLACEMENTS.sql (hunk 1 above)
Comment (luzpaz) [reply projectforge-business/src/main/resources/flyway/migrate/common/V7.4.1.0__7.4.1.0-RELEASE-VACATION_OTHER_REPLACEMENTS.sql]: Apologies. I've removed the file from the commit.

**Review by kreinhard — APPROVED** [review]

**Pull request merged**